pyproject.toml

```diff
@@ -10,6 +10,14 @@ build-backend = "setuptools.build_meta"
 [tool.black]
 line-length = 120
 
+[tool.cibuildwheel.linux]
+archs = ["auto64"]
+skip = ["*musllinux*"]
+before-all = "ln -s /usr/lib64/libgfortran.so.5 /usr/lib64/libgfortran.so.3"
+
+[tool.cibuildwheel.macos]
+repair-wheel-command = ""
```

Comment:
Do we need this section? What's the default for `repair-wheel-command`?

Reply:
Default is below

```
[tool.cibuildwheel.macos]
repair-wheel-command = "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel}"
```

However, in `cmd_line` directory, some binary files link to external dependencies outside the repository, and make the repair command fail.

Reply:
I change to 
```
repair-wheel-command = "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel} --ignore-missing-dependencies"
```

`--ignore-missing-dependencies` should work for this situation.